src/test/javamanager.test.ts

```diff
@@ -1,75 +0,0 @@
-/*
```

Comment:
remove tests which now requires too many quirks to
adapt

there are a lot of paths that would need to be handled and a lot of
specific environments to configure. VS Code Java from where the code was
picked doesn't have tests neither.
So proposing to not having test for the special parts :'-( The part of
having a JDK 11 on the path is correctly tested indirectly via other
tests.